Gemfile

```diff
@@ -6,6 +6,10 @@ gem 'simplecov', :require => false
 gem 'pry-nav'
 gem 'pry-rescue'
 
+# optional dependency for more accurate timezone conversion
+gem 'tzinfo', '1.2.5'
+# gem 'tzinfo', '2.0.0'
```

Comment:
Shouldn't there be changes to the lock file?
Can we use the newer version or at least remove the commented out version?

Reply:
`Gemfile.lock` is actually in the gitignore. For some reason upstream they commented out `2.0.0` to go back to `1.2.5` so I just left it as-is.

Reply:
Sounds good.